Makefile

```diff
@@ -82,12 +82,14 @@ BUILD_DIR := build
 # a zipfile of the source code).
 FORKNAME := $(word 2, $(subst :,,$(subst github.com, ,$(shell git remote get-url origin))))
 FORKUSER := $(word 1, $(subst /, ,$(FORKNAME)))
+RELEASES_BUCKET := gs://releases.naturalcapitalproject.org
+DEV_BUILD_BUCKET := gs://natcap-dev-build-artifacts
```

Comment:
Were these just moved to their own variables for readability and style? I couldn't see if these variables end up being used elsewhere...

Reply:
Yeah, I separated these out in `Makefile` so that they can be referred to directly in `scripts/release-3-publish.sh`:

```shell
# assume a default user of "natcap" if none has been provided.
GH_USER="${2:-'natcap'}"
if [ "$GH_USER" = "natcap" ]
then
    BUCKET="$(make jprint-RELEASES_BUCKET)/invest"
else
    BUCKET="$(make jprint-DEV_BUILD_BUCKET)/invest/$GH_USER"
fi
```

I added a comment to `Makefile` in 10f9df17

Reply:
Ah I see, thanks.